src/Gallifrey.UI.Modern/MainViews/MainWindow.xaml.cs

```diff
@@ -339,31 +339,13 @@ private void MainWindow_KeyUp(object sender, KeyEventArgs e)
                 switch (key)
                 {
                     case Key.A: trigger = RemoteButtonTrigger.Add; break;
+                    case Key.C: trigger = RemoteButtonTrigger.Copy; break;
                     case Key.D: trigger = RemoteButtonTrigger.Delete; break;
                     case Key.F: trigger = RemoteButtonTrigger.Search; break;
                     case Key.E: trigger = RemoteButtonTrigger.Edit; break;
-                    case Key.S: trigger = RemoteButtonTrigger.Export; break;
+                    case Key.U: trigger = RemoteButtonTrigger.Export; break;
                     case Key.L: trigger = RemoteButtonTrigger.LockTimer; break;
-                    case Key.P: trigger = RemoteButtonTrigger.Settings; break;
-                    case Key.I: trigger = RemoteButtonTrigger.Info; break;
-                    case Key.T: trigger = RemoteButtonTrigger.Twitter; break;
-                    case Key.M: trigger = RemoteButtonTrigger.Email; break;
-                    case Key.G: trigger = RemoteButtonTrigger.GitHub; break;
-                    case Key.H: trigger = RemoteButtonTrigger.Gitter; break;
-                    case Key.C: trigger = RemoteButtonTrigger.PayPal; break;
-                    default: return;
-                }
-            }
-            else if (Keyboard.IsKeyDown(Key.LeftAlt) || Keyboard.IsKeyDown(Key.RightAlt))
-            {
-                switch (key)
```

Comment:
Can we  leave the ALT + F(num) shortcuts in please.

Agree with removing the CTRL + Letter ones :)